sklearn/datasets/base.py

```diff
@@ -255,14 +255,25 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
+    if return_X_y == false
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    if return_X_y == true
```

Comment:
i downloaded the PR myself and built the docs, and it looks like this (which seems correct, bar a few nitpicks) to me
<img width="1274" alt="screen shot 2016-07-20 at 11 17 10 am" src="https://cloud.githubusercontent.com/assets/7272031/16997764/a69ff8ec-4e6b-11e6-9a0a-02cd594bd96d.png">